packages/@core.keystone/gen.gql.react.utils.js

```diff
@@ -5,39 +5,25 @@ import { useIntl } from '@core/next/intl'
 const getObjects = (objectsContainer, converter) => (objectsContainer && objectsContainer.objs) ? objectsContainer.objs.map(converter) : []
 
 function genReactHooks (TestUtils, { convertGQLItemToUIState, convertUIStateToGQLItem, options } = { options: {} }) {
-    function useObject (variables, memoize = true) {
-        const { loading, refetch, objs, count, error } = useObjects(variables, memoize)
+    function useObject (variables) {
+        const { loading, refetch, objs, count, error } = useObjects(variables)
         if (count && count > 1) throw new Error('Wrong query condition! return more then one result')
         const obj = (objs.length) ? objs[0] : null
         return { loading, refetch, obj, error }
     }
 
-    function useObjects (variables = {}, memoize = true) {
+    function useObjects (variables = {}, queryOptions = { fetchPolicy: 'no-cache', notifyOnNetworkStatusChange: true }) {
```

Comment:
please don't use no-cache by default

Reply:
It's ok to use no-cache on some page not by default

Reply:
notifyOnNetworkStatusChange - is an important flag to make refetch method trigger loading indicator. 
Without using 'no-cache' table sorting will not render in table as loading indicator will not trigger 
I removed 'no-cache' from default value - but we will need to write it every time we use useObjects together with useTable